torch/csrc/jit/ir/alias_analysis.cpp

```diff
@@ -48,6 +48,7 @@ class MutableTypePtrHelper {
       case TypeKind::DictType:
       case TypeKind::ClassType:
       case TypeKind::TensorType:
+      case TypeKind::UnionType:
```

Comment:
@eellison What's the possible consequence of landing this line? Is it a correctness issue or just perf regression? I learned from Ansley that this is temporary for this PR only and will be fixed by later PRs in the stack.

Reply:
correctness issue